server/db/models/borrowed_books.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+export default (sequelize, DataTypes) => {
+  const Borrowed_books = sequelize.define('Borrowed_books', {
```

Comment:
Borrowed_books???
Stick to camel case. "BorrowedBooks"

Reply:
ok